pytorch_lightning/metrics/functional/classification.py

```diff
@@ -726,92 +726,6 @@ def dice_score(
     return reduce(scores, reduction=reduction)
 
 
-def iou(
```

Comment:
add mock/wrap func with a warning about refactoring

Reply:
Sorry, I do not know how to do that. Can you show me an example?

Reply:
see the example here:
https://github.com/PyTorchLightning/pytorch-lightning/blob/1e501f0f40c873369f6d47214e2471c53311443a/pytorch_lightning/metrics/functional/classification.py#L878
and test deprecation over here:
https://github.com/PyTorchLightning/pytorch-lightning/blob/b50ad9ee95b0c08b644f43e6c9bc8cf34ac8f324/tests/deprecated_api/test_remove_1-2.py#L41